packages/devextreme/js/__internal/core/utils/m_common.ts

```diff
@@ -7,6 +7,9 @@ import {
   isDefined, isFunction, isObject, isString, type,
 } from '@js/core/utils/type';
 
+// @ts-expect-error new deferred
+export const waitUntilFirstWidgetCreating = new Deferred();
```

Comment:
```suggestion
export const firstComponentCreating = new Deferred();
```
or better smth generalized, maybe to highlight the logical segregation of "core/components" layers. Like this:
```suggestion
export const componentsCreating = new Deferred();
```
or
```suggestion
export const componentsLayerInitializing = new Deferred();
```
or
```suggestion
export const uiLayerInitializing = new Deferred();
```
or 
```suggestion
export const domLayerInitializing = new Deferred();
```

etc

Reply:
I suggest renaming it:

```suggestion
export const firstComponentCreated = new Deferred();
```
or better smth generalized, maybe to highlight the logical segregation of "core/components" layers. Like this:
```suggestion
export const componentsCreated = new Deferred();
```
or
```suggestion
export const componentsLayerInitialized = new Deferred();
```
or
```suggestion
export const uiLayerInitialized = new Deferred();
```
or 
```suggestion
export const domLayerInitialized = new Deferred();
```

etc